docs/src/dedicated-gen-2/overview.md

```diff
@@ -0,0 +1,53 @@
+Dedicated Generation 2 works nearly identically to Grid environments, and does not require additional configuration on your part. The only difference is that of service availability.
+
+## Why a Generation 2?
+
+Even though our existing [dedicated infrastructure](/dedicated/overview) ensures better uptime and availability for your applications and services, configuration or application topology changes have to go through our Customer Success team, as the VMs are provisioned by us. Generation 2 gives you both the high availability from our Dedicated offer and the self-service flexibility and features of the Grid. This means that you will now be able to edit your configuration yourself – as you would do on the Grid in your `.platform/services.yaml` and `.platform.app.yaml` files – and see those changes in your Dedicated Generation 2 environments when pushed without having to open a ticket.
+
+## Features
+
+* Domains can be added to your development environments, such as to a dedicated staging environment.
+* Three-Virtual Machine for your master branch.
+* Backups on Production do not freeze in the process.
+* You can deploy your application in any of our [supported cloud providers](/development/faq.md#which-geographic-zones-does-platformsh-cover).
+
+## Discrepancies with the Grid
+
+### Available services
+
+As of today, the following services are available on Dedicated Generation 2:
+- [Mariadb/Mysql](/configuration/services/mysql)
+- [Redis](/configuration/services/redis)
+- [Elasticsearch](/configuration/services/elasticsearch)
+- [RabbitMQ](/configuration/services/rabbitmq)
+- [Solr (6 only)](/configuration/services/solr)
+- [Network storage (2.0 only)](/configuration/services/network-storage)
+
+### Local mounts
+
+Because you get a redundant infrastrcture, note that local mounts will be local to each Virtual Machine. Since you can't know what VM is going to handle a specific request, you don't have a guarantee regarding which local mount is going to be used. Whether you actually want to use a local mount or in fact need to set up a network storage mount depends on your specific use-cases.
+
+## Setting up
+
+### Coming from the Grid
+
+Open a Support ticket with the suggested title “Upgrade to Dedicated Generation 2". In the ticket, request to have your project upgraded.
+
+We will review the infrastructure and settings, create user and environment variables for Staging and Production environments, and update the ticket with those changes.
+
+Once started the process usually takes less than an hour.  There should be no downtime on your production site, although you should not push any code to Git while the upgrade is in progress.
+
+When done, you can review the changes to your project through the management console.
+
+### Coming from Dedicated
+
+Migrating from Dedicated to Dedicated Generation 2 is not possible for now but feel free to reach out so we're aware of the demand.
```

Comment:
Change this to something like... 

Please contact us if you are interested in migrating to Dedicated Generation 2. (and then at that point we can tell customers we if we’re not quite ready etc…)

Reply:
My concern here is that if we make it sound like it can be done now, then getting a "no" is a bad surprise, as we could have just said it upfront.
It looks like supporting Dedicated to Dedicated Generation 2 migrations is going to take a while.

I don't feel too strongly about this but thinking about those who will have to handle the contacts and have these conversations with our customers :).